app/views/content/funding-and-support/tuition-fees.md

```diff
@@ -35,11 +35,19 @@ expander:
       </p>
       <p>
       Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.</p>
+inset_text:
+  fee-increase:
+    text: |-
+      In autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses increased. These are the fees shown on this page. 
```

Comment:
```suggestion
      Since autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses has been increased. The increased amounts are shown on this page. 
```

Reply:
@gemmadallmandfe is this clearer?

Reply:
Would `From autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses has been increased. The increased amounts are shown on this page.` be slightly better?